### PR TITLE
docs: write the readme

### PR DESCRIPTION
**Pull request by thesayyn**
Our readme did not help users coming to this repo, hopefully it's more clear what rules_distroless does.

### CODE REVIEW

**Comment by thesayyn** [comment]
@loosebazooka a look please?

**Comment by loosebazooka** [comment]
oh I didn't even see this when I made #52

File: README.md [hunk 1]
```diff
@@ -1,3 +1,58 @@
...
+- [keytool](/examples/java_keystore)
+- [apt-get install](/examples/debian_snapshot) <i>from Debian repositories.</i>
+- [apt-get install](/examples/ubuntu_snapshot) <i>from Ubuntu repositories.</i>
+
+We also we have distroless-specific rules that could be useful 
+
+- [flatten](/examples/flatten): <i>flatten multiple `tar` archives.</s>
+- [os_release](/examples/os_release): <i>create a `/etc/os-release` file</s>
+- [locale](/examples/locale): <i>strip `/usr/lib/locale` to be smaller.</s>
+- [dpkg_statusd](/examples/statusd): <i>creates a package database at /var/lib/dpkg/status for scanners to discover installed packages.</i>
```
Comment (loosebazooka) [inline README.md:27]:
```suggestion
- [dpkg_statusd](/examples/statusd): <i>creates a package database at /var/lib/dpkg/status.d for scanners to discover installed packages.</i>
```

**Review by loosebazooka — APPROVED** [review]
Looks good, minor fix

File: README.md (hunk 1 above)
Comment (loosebazooka) [reply README.md:27]: or is this something that is distroless specific only (modified for distroless?)

Comment (thesayyn) [reply README.md]: We support both formats though, distroless specific one and the apt generates. so this is fine

**Review by loosebazooka — APPROVED** [review]

**Comment by thesayyn** [comment]
> This branch cannot be rebased due to too many changes 

what in the hell? :(

**Comment by thesayyn** [comment]
@loosebazooka can you force merge? github seems to be stuck

**Comment by loosebazooka** [comment]
I didn't have to force it?

**Comment by thesayyn** [comment]
> I didn't have to force it?

it might have been stuck for me only then. Thanks!

**Pull request merged**